### ISSUE TITLE
remembering filter states in Level 2 scores

### ISSUE THREAD

**Issue by jkapusci** [sul-cidr/noh#325]
When reading Level 2 analysis a user chooses a subset of information within the score (ex. dance only) that selection should be maintained when they go to the next or other shodan.

**Comment by csbailey5t** [comment]
Probably use sessionStorage (unless we rearchitected to have play and section apps within single app)

**Comment by jkapusci** [comment]
Dear Scott, Simon and Peter,
So great to see that you have closed some issues already! Can’t wait to see them active.

I tried to pull the changes thinking that I might be able to preview them on my computer but I ended up with multiple merge conflicts.
I am not sure how to proceed. Sourcetree shows 15 pulls available.
When would any of you be available for a brief phone call to guide me through clearing this?
Thanks!
Jarek

***
Jaroslaw Kapuscinski
Associate Professor
Chair, Department of Music
Stanford University
tel. (650) 723 2717

On Jul 2, 2019, at 7:57 PM, Scott Bailey <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #325<https://github.com/sul-cidr/noh/issues/325> via #356<https://github.com/sul-cidr/noh/pull/356>.
